### ISSUE TITLE
Can this version be installed on any linux?

### ISSUE THREAD

**Issue by egorky** [TheBlode/mDial#1]
Hi, I came from the Vicidial Forums. Does this works only on Opensuse or can be installed on other flavors too? How do I install it? How can I use the ChatGPT integration? Can this be used like a theme on top of the original ViciDial like this one https://github.com/masterfermin02/vici-admin-them ?
:-)

**Comment by kingastersk1** [comment]
Yes, you can install this VICIdial version on Linux, including Ubuntu, and the admin and agent Dashboard can be themed and customized as needed.

**Comment by egorky** [comment]
wow! is there some kind of installation instructions, maybe?

**Comment by kingastersk1** [comment]
This setup follows a standard VICIdial installation (Ubuntu/Linux supported).  
After installing VICIdial, the UI can be deployed on top as a customizable layer.

If you need guidance for installation or customization, feel free to open an issue or contact us via our GitHub profile.
